milatools/cli/commands.py

```diff
@@ -684,6 +687,94 @@ def _standard_server(
         proc.kill()
 
 
+def _get_disk_quota_usage(
+    remote: Remote, print_command_output: bool = True
+) -> tuple[tuple[float, float], tuple[int, int]]:
+    """Checks the disk quota on the $HOME filesystem on the mila cluster.
+
+    Returns whether the quota is exceeded, in terms of storage space or number of files.
+
+    Here is what the output of `disk-quota` looks like on the Mila cluster:
+    ```console
+
+    Quota information for storage pool Default (ID: 1):
+
+          user/group     ||           size          ||    chunk files
+         name     |  id  ||    used    |    hard    ||  used   |  hard
+    --------------|------||------------|------------||---------|---------
+          normandf|1471600598||   97.20 GiB|  100.00 GiB||   806898|  1000000
+    ```
+    """
+    disk_quota_output = remote.get_output("disk-quota", hide=not print_command_output)
+    last_line_parts = disk_quota_output.splitlines()[-1]
+    (
+        _username,
+        _id,
+        _,
+        used_gb,
+        max_gb,
+        _,
+        used_files,
+        max_files,
+    ) = last_line_parts.split("|")
+    used_gb = float(used_gb.removesuffix("GiB").strip())
+    max_gb = float(max_gb.removesuffix("GiB").strip())
```

Comment:
`str.removesuffix()` has been added in python 3.9 I believe which is higher than the minimum stated in the `pyproject.toml`. Are we ok with bumping the minimal version to 3.9? I personally don't have any objections as mila tools is not likely to become a python packages dependency in any project.

Reply:
Fixed in 1b6a5092ebbf60b07ff687bb548c7ab87a64c9fb